browser/util.js

```diff
@@ -15,56 +15,23 @@ import * as config from './config.js';
  *     frameworks have loaded.
  */
 export function whenFrameworksReady(callback) {
-  debug('whenFrameworksReady');
-  var done = function() {
-    debug('whenFrameworksReady done');
+  // webcomponents-loader.js will eventually fire WebComponentsReady.
+  // If WebComponents is not defined and we cannot find the script,
+  // invoke the callback.
+  if (!window.WebComponents &&
+    !document.querySelector('script[src*="webcomponents-loader.js"]')) {
```

Comment:
Re es5/es6 loaders: the idea is to have only `webcomponents-loader.js` (see more info [here](https://github.com/webcomponents/webcomponentsjs/issues/737#issuecomment-288191881)).

`webcomponents-lite.js` and `webcomponents.js` both set `window.WebComponents` and fire `WebComponentsReady` event.

Here we need a way to know if to wait for the WebComponentsReady event or not. Another approach is to always wait for the event for a maximum of 1 second (or any other arbitrary time) and then execute the tests, but that's not any better than this :/